time-clock/db/migrate/20211222000227_create_admins.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAdmins < ActiveRecord::Migration[6.1]
+  def change
+    create_table :admins do |t|
+      t.string :user
+      t.string :password
```

Comment:
Password should not be saved as plain text, pls consider using something like "Devise" for the authentication
It is a classic gem for getting this feature to work without a lot of effort and it has battle tested code. It has probed to be secure a good start point